ui/src/main/resources/FileManagerCode/DriveSheet.xml

```diff
@@ -1832,7 +1832,14 @@ require(['jquery', 'angular', 'JobRunner', 'angular-resource', 'angular-ui-boots
           if (window.confirm("$services.localization.render('fileManager.action.deleteFilesConfirmation')")) {
             // Clear the selection.
             scope.selection = {};
-            scope.drive['delete'](getPaths(files)).done(refreshLiveTable);
+            scope.drive['delete'](getPaths(files)).done(function(result) {
```

Comment:
Is there a message displayed if the delete HTTP request fails due to some network problem? I see we have a callback only for ``done`` here and not for ``fail``, but maybe the ``delete`` method (from ``scope.drive``) does this.

Reply:
As far as I could reproduce it: no.
I had an artificial reproduction method (which I committed by error) where the delete call was preceded by a throw and that always returned a done.

I can imagine some situations where the network goes bad (e.g. pull the plug): Should we also care for this?

Reply:
Let's keep it like this for now.